modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/package.json

```diff
@@ -0,0 +1,40 @@
+{
+	"dependencies": {
+		"@clayui/autocomplete": "3.28.0",
+		"@clayui/button": "3.6.0",
+		"@clayui/color-picker": "3.28.0",
+		"@clayui/data-provider": "3.28.0",
+		"@clayui/drop-down": "3.28.0",
+		"@clayui/form": "3.28.0",
+		"@clayui/icon": "3.1.0",
+		"@clayui/modal": "3.28.0",
+		"@clayui/slider": "3.25.1",
+		"@liferay/frontend-js-react-web": "*",
+		"classnames": "2.2.6",
+		"d3": "5.9.3",
+		"frontend-js-web": "*",
+		"prop-types": "15.7.2"
+	},
+	"jest": {
+		"collectCoverageFrom": [
```

Comment:
Is this property duplicated from [`jest.config.js`](https://github.com/liferay-frontend/liferay-portal/pull/1162/files#diff-86ccda6529a10f5632f09951c8086b3f48ac07abe0001fdf7ff2826cb145366dR14)?

Reply:
yes, i just reviewed this part